### PR TITLE
Gstreamer: 1.14.0 -> 1.14.2

### PR DESCRIPTION
**Pull request by bkchr**
###### Motivation for this change

I could not run `nox-review wip`, as the resulting `nix-build` used more than 20GB of RAM...

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/gstreamer/vaapi/default.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 stdenv.mkDerivation rec {
   name = "gst-vaapi-${version}";
-  version = "1.14.0";
+  version = "1.14.2";
```
Comment (vcunat) [inline pkgs/development/libraries/gstreamer/vaapi/default.nix]: The hash also needs an update.

Comment (bkchr) [reply pkgs/development/libraries/gstreamer/vaapi/default.nix]: Thanks for finding this mistake. Solved :)

Comment (vcunat) [reply pkgs/development/libraries/gstreamer/vaapi/default.nix]: I just tried to build `gst_all_1`... all the rest had succeeded.

Comment (bkchr) [reply pkgs/development/libraries/gstreamer/vaapi/default.nix]: But now it works?

**Comment by vcunat** [comment]
@GrahamcOfBorg build gst_all_1

**Comment by bkchr** [comment]
@vcunat will you merge this?

**Comment by matthewbauer** [comment]
Should go in staging i think.

**Comment by bkchr** [comment]
@matthewbauer is `staging-next` the correct one?

**Comment by matthewbauer** [comment]
Yep that should work

**Comment by LnL7** [comment]
Unless it's a security fix, changes should go to staging and not directly staging-next.

**Comment by vcunat** [comment]
Yes, things are merged to staging instead, at least generally.

**Comment by bkchr** [comment]
Is there any documentation, which pull request should end in which branch? I find it very confusing.

**Comment by vcunat** [comment]
There's an RFC only so far: https://github.com/NixOS/rfcs/pull/26 but that part about where to merge seems to have worked well in the past few months that we're trying the workflow.

**Comment by bkchr** [comment]
Okay, ty :)

**Pull request merged**